src/tools/generate_insights.py

```diff
@@ -102,9 +102,11 @@ class InsightResponse(BaseModel):
 1. One chart insight with appropriate chart type, Recharts-compatible data, and clear axis fields
 2. 1 or 2 simple follow ups to the user query based on the actual data available
 
+Your capabilities: You can analyze data for any area of interest, pull data from datasets like DIST-ALERT, Global Land Cover, Tree Cover Loss, and Grasslands for different time periods, and create charts/insights. Base follow-ups on what's actually possible with available data and tools.
```

Comment:
This should be dynamic if possible, otherwise it will get out of date as we add more datasets.